tests/autogrouping/test_similarity.py

```diff
@@ -0,0 +1,111 @@
+# test it
+import unittest
+import numpy
+import total_scattering.autogrouping.similarity as similarity
+
+
+class TestSimilarity(unittest.TestCase):
+
+    # return r'th component of a triangle weight function extending L indices
+    # in each direction from the "origin". Add 1 to L to account for the
+    # zero-offset ( r == 0 ) weight value
+    def weight_triangular(self, r, L):
+        return (L - abs(r) + 1) / (L + 1)
+
+    '''
+    Weighted cross correlation of vectors within a small window:
+    sum_over_r( weight[ r ] * sum_over_x ( f( x ) * g( x + r ) ) ) for r in
+    union([ 0, L ] and [ 0, -L ] ), inclusive. By default, the weight function
+    is the triangular weight function defined above. Tested with a known
+    analytic solution calculated using numpy functions
+    '''
```

Comment:
Do you mind to move the above comment so that it is a docstring under the `test_weighted_cross_corr` function?